queries/json/folds.scm

```diff
@@ -0,0 +1,3 @@
+[
+  (object)
```

Comment:
Are there nodes for `tables` too ?

Reply:
I think I missed the array, will check it out in a minute

Reply:
ok I've added the `array` and I couldn't find any `tables` node.